code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -167,7 +167,7 @@
 		return FALSE
 	if(!traitor.uplink_handler)
 		return FALSE
-	if(traitor.uplink_handler.progression_points < required_total_progression_points)
+	if(traitor.uplink_handler.progression_points / 60 < required_total_progression_points)
```

Comment:
I don't understand where this number comes from

And considering it comes up in 3 places, and was initially wrong the first, you should make this a define

Reply:
1 MINUTE is 600, not 60. Something like `PROGRESSION_POINT_FACTOR` or similar, with this documentation would be good.